kong/cmd/utils/env.lua

```diff
@@ -0,0 +1,35 @@
+local pl_utils = require "pl.utils"
+local log = require "kong.cmd.utils.log"
+local fmt = string.format
+
+
+local cmd = [[ printenv ]]
```

Comment:
Maybe there is no need for this extra variable anymore, it is used only once.

Reply:
It is used for the debug log as well.